src/NetworkTest/types/opentok/index.ts

```diff
@@ -102,7 +101,6 @@ export namespace OT {
   export type SubscriberProperties = OTSubscriber.SubscriberProperties;
   export type TrackStats = OTSubscriber.TrackStats;
   export type OTError = OTError.OTError;
-  export type PublisherRtcStatsReportArr = OTPublisher.PublisherRtcStatsReportArr;
+  export type PublisherStats = OTPublisher.PublisherStats;
```

Comment:
Just curious: why aren't we using the `opentok.d.ts` file from the OpenTok Node module?